web/themes/custom/my_awesome_theme/components/03-organisms/site/site-footer/_site-footer.scss

```diff
@@ -16,30 +13,62 @@
   }
 }
 
-.footer__social {
-  margin-bottom: $space;
-
-  @include large {
-    flex: 0 1 30%;
-    margin-bottom: 0;
-    margin-right: $space;
-  }
+.footer:before {
+  background-image: url('https://demo.tagdiv.com/newspaper_pro/wp-content/uploads/2021/06/footer-bg1.jpg');
+  content: '';
+  width: 100%;
+  position: absolute;
+  height: 100%;
+  top: 0;
+  left: 0;
+  display: block;
+  z-index: 0;
+  opacity: 0.1;
+  background-size: cover;
+  background-position: center top;
 }
 
 .footer__menu {
   flex: 1 1 100%;
   display: flex;
   flex-wrap: wrap;
-  nav{
+  .contextual-region{
+    .social-list{
+      ul{
+        .item{
+          border: 1px solid rgba(255, 255, 255, 0.03);
+          width: 39.2px;
+          height: 39.2px;
+          margin: 5px 10px 5px 0;
+          background: rgba(255, 255, 255, 0.03);
+        }
+      }
+    }
+  }
+  .views-element-container {
+    div {
+      .js-view-dom-id-4ab2e216877598a5b152e4836b991a2beb7b216c4edb8b3b6ae38c92034f75d8 {
```

Comment:
This selector looks like something that could change at any time.
Investigate if there is something better.

Things like AJAX pager on the view likely will use another random id for every page change.